frontend/src/model/DraftNewReleaseDialog.tsx

```diff
@@ -117,10 +130,13 @@ export default function DraftNewReleaseDialog({ open, handleClose }: { open: boo
           <Button
             variant='contained'
             type='submit'
-            disabled={!semanticVersion || !artefact || !releaseNotes || !releaseName || !isValidSemver(semanticVersion)}
+            disabled={
+              !semanticVersion || !artefacts || !releaseNotes || !releaseName || !isValidSemver(semanticVersion)
+            }
           >
             Create Release
           </Button>
+          <MessageAlert message={errorMessage} severity='error' linkText='More info' />
```

Comment:
`linkText` is only required if `href` is also being provided. This should be enforce via our MessageAlert props type.
```
type PartialMessageAlertProps =
  | {
      linkText: string
      href: string
    }
  | {
      linkText: never
      href: never
    }

type MessageAlertProps = {
  message?: string
  severity?: AlertProps['severity']
} & PartialMessageAlertProps
```